synthtool/gcp/common.py

```diff
@@ -17,23 +17,30 @@
 import re
 import yaml
 from pathlib import Path
-from typing import List, Dict
+from typing import List, Dict, Optional
 
 from synthtool.languages import node
-from synthtool.sources import templates
-from synthtool import __main__
-from synthtool import _tracked_paths
-from synthtool import metadata
+from synthtool.sources import git, templates
+from synthtool import __main__, _tracked_paths, log, metadata
 
 
-_TEMPLATES_DIR = Path(__file__).parent / "templates"
 _RE_SAMPLE_COMMENT_START = r"\[START \w+_quickstart\w*]"
 _RE_SAMPLE_COMMENT_END = r"\[END \w+_quickstart\w*]"
+_TEMPLATES_URL: str = git.make_repo_clone_url("googleapis/synthtool")
+_LOCAL_TEMPLATES: Optional[str] = os.environ.get("SYNTHTOOL_TEMPLATES")
 
 
 class CommonTemplates:
     def __init__(self):
-        self._templates = templates.Templates(_TEMPLATES_DIR)
+        templates_repo = _LOCAL_TEMPLATES
```

Comment:
I would expect to set the environment variable SYNTHTOOL_TEMPLATES to the full path of the templates directory (including synthtool/gcp/templates.)